python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -180,6 +184,56 @@ def replace_illegal_columns_chars(char: str, database: BaseDatabase) -> str:
 
         return normalize_config
 
+    def get_openlineage_facets(self) -> OpenLineageFacets:
+        """
+        Returns the lineage data
+        """
+        input_files = resolve_file_path_pattern(
+            self.input_file.path,
+            self.input_file.conn_id,
+            normalize_config={},
+            filetype=self.input_file.type.name,
+        )
+
+        input_dataset: list[OpenlineageDataset] = [
+            OpenlineageDataset(
+                namespace=self.input_file.openlineage_dataset_namespace,
+                name=self.input_file.openlineage_dataset_name,
+                facets=InputFileDatasetFacet(
+                    file_size=self.input_file.size,
+                    is_pattern=self.input_file.is_pattern(),
+                    files=[file.path for file in input_files],
+                    number_of_files=len(input_files),
+                    file_type=self.input_file.type.name,
+                ),
+            )
+        ]
+
+        output_dataset: list[OpenlineageDataset] = [OpenlineageDataset(namespace=None, name=None, facets={})]
+        if self.output_table:
+            output_database = create_database(self.output_table.conn_id)
+            output_dataset = [
+                OpenlineageDataset(
+                    namespace=output_database.openlineage_dataset_namespace(),
+                    name=output_database.openlineage_dataset_name(table=self.output_table),
+                    facets=OutputDatabaseDatasetFacet(
```

Comment:
You need to pass this in a dictionary: https://github.com/OpenLineage/OpenLineage/blob/7243d916f5400dbbceaece5cda89da961ad005d3/client/python/openlineage/client/run.py#L54-L53

Reply:
Added it here https://github.com/astronomer/astro-sdk/pull/1025/commits/53013ecbd79bd73aba37f595599cf756f1a5ea01